package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -180,22 +297,107 @@ define([
     //endregion
 
     //region Update
+    /**
+     * Updates the view to match its model's latest state.
+     *
+     * When [isAutoUpdate]{@link pentaho.visual.base.View#isAutoUpdate} is `true`,
+     * any change to the view's model automatically triggers a view update, by calling this method.
```

Comment:
any change to the view's model automatically triggers a view update by calling this method.